src/client/java/teammates/client/scripts/sql/VerifyCourseEntityAttributes.java

```diff
@@ -0,0 +1,159 @@
+            List<Student> newSectionStudents = sectionToNewStuMap.get(section.getName());
+
+            // If sectionStudent is null, then section is not present in sql
+            boolean isSectionNamePresent = oldSectionStudents != null && newSectionStudents != null;
+            if (!isSectionNamePresent) {
+                return false;
+            }
+
+            // Group students by team
+            Map<String, List<CourseStudent>> teamNameToOldStuMap = oldSectionStudents.stream()
+                    .collect(Collectors.groupingBy(CourseStudent::getTeamName));
+            Map<String, List<Student>> teamNameToNewStuMap = newSectionStudents.stream()
+                    .collect(Collectors.groupingBy(Student::getTeamName));
+            return verifyTeams(section, teamNameToOldStuMap, teamNameToNewStuMap);
+        });
+
+    }
+
+    private boolean verifyTeams(Section newSection,
+            Map<String, List<CourseStudent>> teamNameToOldStuMap, Map<String, List<Student>> teamNameToNewStuMap) {
+
+        List<Team> newTeam = newSection.getTeams();
+
+        boolean isTeamCountEqual = newTeam.size() != teamNameToNewStuMap.size()
+                || newTeam.size() != teamNameToOldStuMap.size();
+        if (!isTeamCountEqual) {
+            return false;
+        }
+
+        return newTeam.stream().allMatch(team -> {
+            List<CourseStudent> oldTeamStudents = teamNameToOldStuMap.get(team.getName());
+            List<Student> newTeamStudents = teamNameToNewStuMap.get(team.getName());
+
+            // If teamStudents is null, then team is not present in sql
+            boolean isTeamNamePresent = oldTeamStudents != null && newTeamStudents != null;
+            if (!isTeamNamePresent) {
+                return false;
+            }
+            return verifyStudents(oldTeamStudents, newTeamStudents);
+        });
+    }
+
+    private boolean verifyStudents(
+            List<CourseStudent> oldTeamStudents, List<Student> newTeamStudents) {
+        if (oldTeamStudents.size() != newTeamStudents.size()) {
+            return false;
+        }
+        oldTeamStudents.sort((a, b) -> a.getEmail().compareTo(b.getEmail()));
+        newTeamStudents.sort((a, b) -> a.getEmail().compareTo(b.getEmail()));
+        for (int i = 0; i < oldTeamStudents.size(); i++) {
+            CourseStudent oldStudent = oldTeamStudents.get(i);
+            Student newStudent = newTeamStudents.get(i);
+            if (!verifyStudent(oldStudent, newStudent)) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    private boolean verifyStudent(CourseStudent oldStudent,
+            Student newStudent) {
+        return newStudent.getName().equals(oldStudent.getName())
+                && newStudent.getEmail().equals(oldStudent.getEmail())
+                && newStudent.getComments().equals(oldStudent.getComments());
+    }
+
+    private List<Student> getCurrStudents(String courseId) {
```

Comment:
Lets stick to the `old` and `new` terms to make things consistent
```suggestion
    private List<Student> getNewStudents(String courseId) {
```